ghcide/src/Development/IDE/Core/OfInterest.hs

```diff
@@ -94,11 +99,22 @@ kick = do
     ShakeExtras{progressUpdate} <- getShakeExtras
     liftIO $ progressUpdate KickStarted
 
-    -- Update the exports map for the project
+    -- Update the exports map for FOIs
```

Comment:
Expand this out? Files of Interest I assume

Reply:
we have this acronym in several places including datatypes - it's ok

Reply:
Fair enough :)